pkg/utils/addons/addons.go

```diff
@@ -98,18 +111,26 @@ func CreatePipelines(template mf.Manifest, client client.Client) (mf.Manifest, e
 		newTempRes.SetName(name)
 		pipelineParams, found, err := unstructured.NestedFieldNoCopy(newTempRes.Object, "spec", "params")
 		if !found || err != nil {
-			return mf.Manifest{}, err
+			return nil, err
 		}
 
 		tasks, found, err := unstructured.NestedFieldNoCopy(newTempRes.Object, "spec", "tasks")
 		if !found || err != nil {
-			return mf.Manifest{}, err
+			return nil, err
+		}
+		var index int
+		if !resourced {
+			index = 1
+		}
+		taskBuild := tasks.([]interface{})[index].(map[string]interface{})
+		taskName := name
+		if resourced {
+			taskName += "-pr"
```

Comment:
if the default value of `index` is set to 1, then 
the 2 `if` blocks can be combined

Reply:
right, I did this for the other one. Thanks